targets/targets.json

```diff
@@ -4611,6 +4959,8 @@
             "function": "LPCTargetCode.lpc_patch"
         },
         "device_name": "LPC54114J256BD64",
+        "mbed_rom_start": "0x0",
+        "mbed_rom_size": "0x40000",
```

Comment:
We've described `PROGRAM_FLASH`, but shouldn't we also add `mbed_ram*` entries for this device's `SRAM0`, `SRAM1`, `SRAM2`, and `SRAMX`?